addons/binding/org.openhab.binding.hp1000/build.properties

```diff
@@ -0,0 +1,8 @@
+source..=src/main/java/
+output..=target/classes
+bin.includes=META-INF/,\
+             .,\
+             about.html,\
+             OSGI-INF/,\
+             ESH-INF/
+             
```

Comment:
I think ins this line is an invisible space or tab, please remove it so the last line is empty.